dxp/liferay_component.md

```diff
@@ -44,20 +44,35 @@ Liferay.componentReady([...id]).then(function)
 
 A common use case is the need of run some actions after a component which is not in scope is created.
 
-# `Liferay.destroyComponent()`
+# Lifecycle
 
-This is the safest way to destroy a registered component. It destroys the component registered with the provided component ID. This invokes the component's own destroy lifecycle methods (`destroy` or `dispose`) and deletes the internal references to the component in the component registry.
+We're offering some OOTB lifecycles so you don't need to worry about registering / destroying the components all the time.
```

Comment:
"We're offering" → "We offer"

I'd spell out "out-of-the-box" instead of OOTB, in case people don't know what it means.

You could also drop "the components" to just "components".